### PR TITLE
Feat: Restaurant model added to database

### PR DESCRIPTION
**Pull request by Danrod16**
Screenshot 2022-05-30 at 10 45 43
Finsihed restaurant models and front end for devise signup

### CODE REVIEW

File: app/models/restaurant.rb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+class Restaurant < ApplicationRecord
```
Comment (Danrod16) [inline app/models/restaurant.rb:1]: You forgot validations

Comment (Danrod16) [reply app/models/restaurant.rb:1]: yeah you are right

**Comment by jYannni** [comment]
"You forgot Validations" - as stated by Daniel

**Review by Danrod16 — COMMENTED** [review]
Approves

**Review by jYannni — COMMENTED** [review]
Yanko Approves too

**Pull request merged**